app/addons/documents/index-results/index-results.components.react.jsx

```diff
@@ -53,10 +53,22 @@ function (app, FauxtonAPI, React, Stores, Actions, Components, Documents) {
         </a>);
     },
 
+    getRevTreeUrlFragment: function (url) {
+      return (
+        <a href={url}>
+          <i className="fonticon-sitemap"></i>
+        </a>);
+    },
+
     getDocumentList: function () {
       var noop = function () {};
 
       return _.map(this.props.results, function (doc, i) {
+
+        var splitArr = (doc.url).split("/");
+        var docID = JSON.parse(doc.content)._id;
+        var winner = JSON.parse(doc.content)._rev;
```

Comment:
i would add this to the store to the doc object in the store (in getResults, getMangoDoc)